etcd/src/test/java/org/jclouds/etcd/features/KeysApiLiveTest.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jclouds.etcd.features;
+
+import static org.testng.Assert.assertNotNull;
+import static org.testng.Assert.assertNull;
+import static org.testng.Assert.assertTrue;
+
+import java.util.UUID;
+
+import org.jclouds.etcd.BaseEtcdApiLiveTest;
+import org.jclouds.etcd.domain.keys.Key;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import com.google.common.base.Throwables;
+
+@Test(groups = "live", testName = "KeysApiLiveTest")
+public class KeysApiLiveTest extends BaseEtcdApiLiveTest {
+
+   private String key;
+   private String value;
+
+   @BeforeClass
+   protected void init() {
+      key = UUID.randomUUID().toString().replaceAll("-", "");
+      value = UUID.randomUUID().toString().replaceAll("-", "");
```

Comment:
Just seen that this "key id formatting convention" is applied everywhere. Better create a static helper `randomKey()` to generate the key ids?